samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/spotlight/SpotlightExampleNode.kt

```diff
@@ -25,27 +38,27 @@ import com.bumble.appyx.core.node.Node
 import com.bumble.appyx.core.node.ParentNode
 import com.bumble.appyx.navmodel.spotlight.Spotlight
 import com.bumble.appyx.navmodel.spotlight.backpresshandler.GoToPrevious
+import com.bumble.appyx.navmodel.spotlight.current
 import com.bumble.appyx.navmodel.spotlight.elementsCount
 import com.bumble.appyx.navmodel.spotlight.hasNext
 import com.bumble.appyx.navmodel.spotlight.hasPrevious
-import com.bumble.appyx.navmodel.spotlight.operation.next
 import com.bumble.appyx.navmodel.spotlight.operation.activate
+import com.bumble.appyx.navmodel.spotlight.operation.next
 import com.bumble.appyx.navmodel.spotlight.operation.previous
 import com.bumble.appyx.navmodel.spotlight.operation.updateElements
-import com.bumble.appyx.navmodel.spotlight.transitionhandler.rememberSpotlightSlider
 import com.bumble.appyx.sandbox.client.child.ChildNode
-import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.Item.C1
-import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.Item.C2
-import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.Item.C3
 import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.NavTarget.Child1
 import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.NavTarget.Child2
 import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.NavTarget.Child3
-import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.State.Loaded
-import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.State.Loading
+import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.ScreenState.Loaded
+import com.bumble.appyx.sandbox.client.spotlight.SpotlightExampleNode.ScreenState.Loading
 import kotlinx.coroutines.delay
 import kotlinx.coroutines.launch
 import kotlinx.parcelize.Parcelize
 
+/**
+ * Shows how to use spotlight to create a UI with BottomTabBar
```

Comment:
Does it make sense that this is a BottomTabBar sample? Why not put this in the sample app instead of sandbox?

Reply:
Hmm, not sure to be honest. This demo was initially placed in sandbox so I decided to leave it there for now